src/components/BookList.jsx

```diff
@@ -0,0 +1,27 @@
+function BookList() {
+  const books = [{
+    title: 'pride and prejudice',
+    author: 'jane austen',
+  }, {
+    title: 'unleash man within',
+    author: 'sathiya sam',
+  }, {
+    title: 'Man with a plan',
+    author: 'Matt Le Blanc',
+  }, {
+    title: 'Breaking Bad',
+    author: 'Vince Gilligan',
+  }];
+
+  return (
+    <div className="bookList container">
+      {books.map((book) => (
+        <div key="book">
+          <h3>{book.title}</h3>
+          <p>{book.author}</p>
+        </div>
+      ))}
+    </div>
```

Comment:
- [ ] Each book should have a button for removing the book, kindly add it this as it is a requirement. 
#### Please not functionality is not expected just the component